### PR TITLE
chore: fix core vsix on windows

### PR DESCRIPTION
**Pull request by gbockus-sf**
### What does this PR do?
Add the older jsforce as a dep of the core extension.  This is a sad work around to the issue found where VSCode 1.83.1 isn't loading modules correctly

Downsides: 
- one more install of jsforce in the core extension 
- Inflates the core extension by another 5mb or so

### What issues does this PR fix or reference?
@W-14374832@ 
#5187

### Functionality Before
vsix fails to start on windows

### Functionality After
vsix works on windows

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
         "@salesforce/source-tracking": "4.2.14",
         "@salesforce/templates": "^58.0.1",
         "@salesforce/source-deploy-retrieve": "9.7.18",
+        "jsforce": "1.11.0",
```
Comment (gbockus-sf) [inline packages/salesforcedx-vscode-core/package.json:95]: Note this is in the packageUpdates property.  These are deps that are directly installed b/c they can't be bundled.  In this case we are ensuring that the version of sf-core in the @heroku/functions/core dependancy finds the right version of jsforce.

**Review by daphne-sfdc — APPROVED** [review]
Thanks for the workaround Gordon!

**Comment by gbockus-sf** [comment]
ouch

**Comment by gbockus-sf** [comment]
This was tested on other Windows and it appears to be an issue with my env.  Closing

**Pull request closed without merging**